dipy/viz/ui.py

```diff
@@ -2952,6 +2952,283 @@ def set_img(self, img):
         self.texture = set_input(self.texture, img)
 
 
+class Option(UI):
```

Comment:
I was expecting the Option class to have a `select()` and a `deselect()` methods which would take care of changing the option's appearance accordingly.